### PR TITLE
11.0 mig purchase all shipments

### PR DESCRIPTION
**Pull request by mpanarin**
Migrate module purchase_all_shipments to v11

### CODE REVIEW

**Review by ghost — APPROVED** [review]
LGTM

File: purchase_all_shipments/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+              "Odoo Community Association (OCA),",
+    "category": "Purchases",
+    "license": "AGPL-3",
+    "depends": [
+        "purchase",
+    ],
+    "data": [
+        "view/purchase_order.xml",
+    ],
+}
```
Comment (MeetKD) [inline purchase_all_shipments/__manifest__.py:20]: Website key is missing and it would be great if you can add a summary as well.

**Review by MeetKD — CHANGES_REQUESTED** [review]

File: purchase_all_shipments/__manifest__.py (hunk 1 above)
Comment (mpanarin) [reply purchase_all_shipments/__manifest__.py:20]: updated

File: purchase_all_shipments/models/purchase_order.py [hunk 2]
```diff
@@ -0,0 +1,47 @@
+# Copyright 2018 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+from odoo import models, fields, api
```
Comment (hieulucky111) [inline purchase_all_shipments/models/purchase_order.py:4]: Order import: https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#imports

**Review by hieulucky111 — APPROVED** [review]
Some minor adjustments.

**Review by nicomacr — APPROVED** [review]

File: purchase_all_shipments/__init__.py [hunk 3]
```diff
@@ -0,0 +1,4 @@
+# Copyright 2018 Camptocamp SA
```
Comment (nikul-serpentcs) [inline purchase_all_shipments/__init__.py:1]: @mpanarin IMO remove Copyright in all __init__.py file.

**Review by nikul-serpentcs — COMMENTED** [review]
Improve Code

**Pull request closed without merging**